pkg/client/client.go

```diff
@@ -261,34 +277,7 @@ func FormatStatusResponse(w io.Writer, sr *models.StatusResponse, allAddresses,
 			fmt.Fprintf(w, "Kubernetes APIs:\t[\"%s\"]\n", strings.Join(sr.Kubernetes.K8sAPIVersions, "\", \""))
 		}
 		if sr.KubeProxyReplacement != nil {
-			features := []string{}
-
-			if np := sr.KubeProxyReplacement.Features.NodePort; np.Enabled {
-				mode := np.Mode
-				if mode == models.KubeProxyReplacementFeaturesNodePortModeHYBRID {
-					mode = strings.Title(mode)
-				}
-				features = append(features,
-					fmt.Sprintf("NodePort (%s, %d-%d, XDP: %s)",
-						mode, np.PortMin, np.PortMax, np.Acceleration))
-			}
-
-			if sr.KubeProxyReplacement.Features.HostPort.Enabled {
-				features = append(features, "HostPort")
-			}
-
-			if sr.KubeProxyReplacement.Features.ExternalIPs.Enabled {
-				features = append(features, "ExternalIPs")
-			}
-
-			if hs := sr.KubeProxyReplacement.Features.HostReachableServices; hs.Enabled {
-				features = append(features, fmt.Sprintf("HostReachableServices (%s)",
-					strings.Join(hs.Protocols, ", ")))
-			}
-
-			if sr.KubeProxyReplacement.Features.SessionAffinity.Enabled {
-				features = append(features, "SessionAffinity")
-			}
+			fmt.Fprintf(w, "KubeProxyReplacement: Status: \t%s\n", sr.KubeProxyReplacement.Mode)
```

Comment:
If the kube-proxy-replacement flag is set to disabled, then it probably doesn't make sense to print any of the subsequent lines?

Reply:
On the other hand, do we want to convey what functionalities are disabled when the `KubeProxyReplacement` mode is set to disabled?

Reply:
The default is probe, right? If the user already made a determination that they do not want `KubeProxyReplacement` and explicitly disabled it, then I'm not sure that we need to advertise to them what they're missing.

Reply:
^ Note that if we are representing this information another way, like a dedicated CLI for services / kube-proxy replacement or via `cilium config` which comprehensively lists all options, then I'm fine with printing everything all the time.